src/Npgsql/NpgsqlTypes/NpgsqlTsQuery.cs

```diff
@@ -49,23 +50,27 @@ public enum NodeKind
             /// <summary>
             /// Lexeme
             /// </summary>
-            Lexeme,
+            Lexeme = 0,
             /// <summary>
             /// Not operator
             /// </summary>
-            Not,
+            Not = 1,
             /// <summary>
             /// And operator
             /// </summary>
-            And,
+            And = 2,
             /// <summary>
             /// Or operator
             /// </summary>
-            Or,
+            Or = 3,
+            /// <summary>
+            /// "Followed by" operator
+            /// </summary>
+            Phrase = 4,
             /// <summary>
             /// Represents the empty tsquery. Should only be used at top level.
             /// </summary>
-            Empty
+            Empty = 5
```

Comment:
Why not leave that job to the compiler?

Reply:
I'm guessing @YohDeadfall wasn't aware that the TsQueryHandler casts NodeKind to byte and writes that to PostgreSQL, so the integral values have meaning. In fact, I'd prefer that all enum values here have their values manually assigned, like you did for `Empty`.

Reply:
And of course I now see that you did that @rwasef1830 :)

Reply:
Indeed, but it looks strange since the integer value of `Empty` is changed.